rollup.config.js

```diff
@@ -0,0 +1,12 @@
+import babel from 'rollup-plugin-babel';
+import pkg from './package.json';
+
+export default {
+	input: 'src/index.js',
+	output: {
+		file: pkg.main,
+		format: 'cjs'
+	},
+	plugins: [babel()],
```

Comment:
Instead of a `.babelrc`, you can include the configuration here directly as an argument to `babel()`.